packages/data-row-edit/package.json

```diff
@@ -3,122 +3,47 @@
   "version": "3.0.5",
   "license": "LGPL-3.0",
   "description": "Data row edit",
-  "author": "sido <sido@haakma.org>",
-  "publishConfig": {
-    "access": "public"
-  },
+  "author": "connor <c.stroomberg@rug.nl>",
```

Comment:
Is het niet handiger om als auteur hier het gcc/umcg neer te zetten?

Reply:
mag van mij , hebben niet echt een standaard voor